mc/query/query_test.go

```diff
@@ -0,0 +1,369 @@
+package query
+
+import (
+	pb "github.com/mediachain/concat/proto"
+	"reflect"
+	"testing"
+)
+
+var simpleq []string = []string{
+	"SELECT * FROM foo.bar",
+	"SELECT body FROM foo.bar",
+	"SELECT id FROM foo.bar",
+	"SELECT source FROM foo.bar",
+	"SELECT timestamp FROM foo.bar",
+	"SELECT (body, source) FROM foo.bar",
+	"SELECT COUNT(*) FROM foo.bar",
+	"SELECT namespace FROM *",
+	"SELECT * FROM foo.bar.*",
+	"SELECT * FROM foo.bar-baz-with-dashes",
+	"SELECT * FROM foo.bar WHERE id = abc",
+	"SELECT * FROM foo.bar WHERE publisher = abc",
+	"SELECT * FROM foo.bar WHERE source = abc",
+	"SELECT * FROM foo.bar WHERE timestamp < 1474000000",
+	"SELECT * FROM foo.bar WHERE timestamp <= 1474000000",
+	"SELECT * FROM foo.bar WHERE timestamp = 1474000000",
+	"SELECT * FROM foo.bar WHERE timestamp >= 1474000000",
+	"SELECT * FROM foo.bar WHERE timestamp > 1474000000",
+	"SELECT * FROM foo.bar WHERE publisher = abc AND timestamp > 1474000000",
```

Comment:
Added != for both timestamp and value criteria. 
NOT is a little trickier as it is a unary prefix operator.